scripts/format/index.mjs

```diff
@@ -0,0 +1,74 @@
+import { readdir, readFile, writeFile } from 'fs/promises';
+import { join, extname } from 'path';
+import chalk from 'chalk';
+import lint from './lint.mjs';
+import space from './space.mjs';
+
+async function main() {
+  // chinese doc use markdownlint and autocorrect to format
+  let zhChanged = await formatDocs('./content/zh/docs/hertz', [
+    function (content) {
+      return space(content);
+    },
+  ]);
+
+  // english doc use markdownlint to format
+  let enChanged = await formatDocs('./content/en/docs/hertz');
```

Comment:
changes only for hertz ?

Reply:
> changes only for hertz ?

The modification folder can contain all documents. Although the changes are small enough (most of them are spaces), it is still necessary to review all changes when opening to all documents. Do I need to include all documents?